### PR TITLE
Add TypeScript keyword, update README with latest mappings

### PR DESCRIPTION
**Pull request by mnkhouri**
I added the TypeScript keyword, with the same values as the Javascript
keyword + "typescript" + "react".

I also updated the README to reflect the latest mappings, including this
change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,48 +53,53 @@ Mappings that are not overridden this way default to what is found in the `DashD
...
-      "Scala"                 : ["scala", "akka", "playscala"],
+      "Scala"                 : ["scala", "akka", "playscala", "scaladoc"],
       "Shell-Unix-Generic"    : ["bash", "manpages"],
       "SQL"                   : ["mysql", "sqlite", "psql"],
       "TCL"                   : ["tcl"],
-      "YAML"                  : ["yaml"]      
+      "TSS"                   : ["titanium"],
+      "TypeScript"            : ["typescript", "javascript", "react", "nodejs", "jquery", "jqueryui", "jquerym", "angularjs", "backbone", "marionette", "meteor", "sproutcore", "moo", "prototype", "bootstrap", "foundation", "lodash", "underscore", "ember", "sencha", "extjs", "knockout", "zepto", "yui", "d3", "svg", "dojo", "express", "mongoose", "moment", "require", "awsjs", "jasmine", "sinon", "grunt", "chai", "html", "css", "cordova", "phonegap", "unity3d", "titanium"],
+      "YAML"                  : ["yaml"],
+      "XML"                   : ["xml","titanium"]
```
Comment (farcaller) [inline README.md]: can you add a missing space after `,` please?

Comment (mnkhouri) [reply README.md]: you betcha @farcaller! Updated

**Comment by farcaller** [comment]
Thanks!

**Comment by farcaller** [comment]
released as 2.4

**Pull request merged**